instrumentation/azure-core/azure-core-1.53/library-instrumentation-shaded/build.gradle.kts

```
@@ -0,0 +1,29 @@
+plugins {
+  id("com.gradleup.shadow")
+  id("otel.java-conventions")
+}
+
+group = "io.opentelemetry.javaagent.instrumentation"
+
+dependencies {
+  // this is the latest version that works with azure-core 1.53
+  implementation("com.azure:azure-core-tracing-opentelemetry:1.0.0-beta.50")
```

sorry, I got confused, I don't think we want this comment after all since `1.0.0-beta.50` is currently the latest version, and possibly there will be later versions that still work with azure-core 1.53

```suggestion
  implementation("com.azure:azure-core-tracing-opentelemetry:1.0.0-beta.50")
```

---

thanks, will fix! there are some legit test failures, looking